processing/data_collection/gazette/spiders/rn_natal.py

```diff
@@ -0,0 +1,46 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider, FormRequest
+from gazette.items import Gazette
+
+
+class RnNatalSpider(Spider):
+    GAZETTE_ELEMENT_CSS = '#texto a'
+    MUNICIPALITY_ID = '2408102'
+
+    allowed_domains = ['www.natal.rn.gov.br']
+    name = 'rn_natal'
+
+    def start_requests(self):
+        base_url = 'http://www.natal.rn.gov.br/dom/'
+
+        for year in range(2015, datetime.now().year + 1):
+            for month in range(1, 13):
+                print([year, month])
```

Comment:
If you want to log something, use `self.logger`. But I don't see the need to log this information here (except for development).